jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -157,7 +157,11 @@ YUI.add('deployment-flow', function() {
       const groupedChanges = this.props.groupedChanges;
       switch (section) {
         case 'model-name':
-          completed = false;
+          completed = this.props.acl.isReadOnly() ||
+            (
+              (this.state.modelName !== 'mymodel' || hasCloud)
```

Comment:
Ah... Basically if someone doesn't change the default name of 'mymodel' we assume that adding a cloud implicitly indicates they're done and happy with the name (they can obviously change it afterwards, but for 'green tick progress' that section is "done").